### DISCUSSION TITLE
How to load fonts from next/font?

### DISCUSSION THREAD

**Discussion by zwwuu** [mantinedev/mantine#3363] [Q&A] [answered]
How to load fonts from next/font? 

[https://nextjs.org/docs/basic-features/font-optimization](https://nextjs.org/docs/basic-features/font-optimization)
[https://www.npmjs.com/package/@next/font](https://www.npmjs.com/package/@next/font)

**Comment by rtivital** [comment]
The way it is described in next.js documentation. What is the issue?

**Comment by AnisRouane67** [comment, marked as answer]
In `_app.js`, at the top of the file, just after the imports, add : 

```
import { Barlow } from "@next/font/google"; // import the font you want from google fonts. If you want custom fonts, see next.js docs
const BarlowFont = Barlow({ weight: "400", subsets: ["latin"] });
```

and then in your MantineProvider, add `fontFamily: BarlowFont.style.fontFamily`

That's it
